streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsProducerTest.java

```diff
@@ -88,44 +88,11 @@ public void before() {
         eosStreamsProducer.initTransaction();
     }
 
-    @Test
```

Comment:
Why do we need to move these tests?

Reply:
I try to keep "order" and group test methods to keep an overview if test coverage is complete.

// generic tests
    // functional
    // exception handling
// non-EOS tests
    // functional
    // exception handling
// EOS tests
    // functional
    // exception handling